content/joysticks/winwing/custom-profiles/_index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Creating custom profiles
+weight: 50
```

Comment:
```suggestion
weight: 30
```

Reply:
Page weight should be `20` so it sorts after the custom profiles and before WINWING CDU pages.